server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -51,19 +45,17 @@ def test_process_tb_file_not_found_error(self, monkeypatch, caplog, test_logger)
         tb_path = fixture_path / "bad_tarball"
 
         def mock_get(self, section: str, option: str):
-            if option == "pbench-receive-dir-prefix":
-                return tb_path
-            if option == "put-token":
-                return "Authorization-token"
-            assert (
-                False
-            ), f"Unexpected configuration option, {section!r}/{option!r}, in mocked config.get()"
+            assert option in ["pbench-receive-dir-prefix", "put-token"]
+            return (
+                tb_path
+                if option == "pbench-receive-dir-prefix"
+                else "Authorization-token"
+            )
 
         def mock_results_push(self, ctrl, tb, token):
             bad_tb = self.receive_dir / bad_tarball
-            if bad_tb == tb:
-                raise FileNotFoundError(f"No such file or directory: '{tb.name}'")
-            assert False, "Unexpected file or directory in mocked results_push"
+            assert bad_tb == tb
```

Comment:
This code is fine, but the expression in line 56 is quite simple, and I think doing it in a single line is clearer (e.g., it prevents a reader like me from wandering off to see if `bad_tb` is a variable from the outer scope...):
```
            assert tb == self.receive_dir / bad_tarball
```
(And, given that the assertion has no explanatory text, that expression is much more self-expressive than the simple comparison of two variables.)